### PR TITLE
Validation Result Common List Items (DEV)

### PR DESCRIPTION
**Pull request by d4rken**
Each validation result screen is a list of several items.
This PR contains placeholders for each item type; to be filled out.

The ViewModels on each screen just have to create a list of items in the right order.

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/common/listitem/ValidationInputVH.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        CovidCertificateValidationResultInputItemBinding.bind(itemView)
+    }
+
+    override val onBindData: CovidCertificateValidationResultInputItemBinding.(
+        item: Item,
+        payloads: List<Any>,
+    ) -> Unit = { item, payloads ->
+
+        val arrivalDateString = item.validation.userInput.arrivalAt.toUserTimeZone().run {
+            "${toShortDayFormat()}, ${toShortTimeFormat()}"
```
Comment (BMItr) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/common/listitem/ValidationInputVH.kt:29]: ⏬ there is no comma between date and time in figma imho
please check

**Comment by BMItr** [comment]
@d4rken 
I kindly ask:
could you make this layout reusable? 
https://github.com/corona-warn-app/cwa-app-android/blob/61795c7004d24d080a5a06002af9131751e5824d/Corona-Warn-App/src/main/res/layout/covid_certificate_validation_failed_fragment.xml

all result screens are very similar.
List is modular anyway, so as far as I see only headerImage & Title is changing.
(as title is already accessible, it would be only changing the name)

Code LGTM!

**Comment by d4rken** [comment]
> @d4rken
> I kindly ask:
> could you make this layout reusable?
> https://github.com/corona-warn-app/cwa-app-android/blob/61795c7004d24d080a5a06002af9131751e5824d/Corona-Warn-App/src/main/res/layout/covid_certificate_validation_failed_fragment.xml

That's from my other draft PR, right?
Sure! Could probably be re-used already, just have to set the title and image, but I'll give it a more common name.
Or what did you have in mind?

**Comment by BMItr** [comment]
> > @d4rken
> > I kindly ask:
> > could you make this layout reusable?
> > https://github.com/corona-warn-app/cwa-app-android/blob/61795c7004d24d080a5a06002af9131751e5824d/Corona-Warn-App/src/main/res/layout/covid_certificate_validation_failed_fragment.xml
> 
> That's from my other draft PR, right?
> Sure! Could probably be re-used already, just have to set the title and image, but I'll give it a more common name.

exactly! :-)

**Review by ChristinaTsigganos — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/common/listitem/ValidationInputVH.kt (hunk 1 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/common/listitem/ValidationInputVH.kt]: comma removed

**Review by jurajkusnier — DISMISSED** [review]

**Review by BMItr — DISMISSED** [review]
please ping after resolving 👌

**Review by jurajkusnier — APPROVED** [review]

**Review by BMItr — APPROVED** [review]

**Pull request merged**